### PR TITLE
test: align usage of jest expect across tests

### PR DESCRIPTION
**Pull request by armano2**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +46,6 @@
     "eslint-plugin-flowtype": "^5.8.0 || ^8.0.2",
     "eslint-plugin-import": "^2.25.3",
     "estraverse": "^5.3.0",
-    "expect": "^24.9.0",
```
Comment (ljharb) [inline package.json:48]: Using the package and the explicit imports are quite intentional; being more coupled to jest is undesirable.

File: __tests__/src/util/isNonInteractiveRole-test.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
-/* eslint-env mocha */
-import expect from 'expect';
+/* eslint-env jest */
```
Comment (ljharb) [inline __tests__/src/util/isNonInteractiveRole-test.js:1]: This should really be done with overrides in eslintrc, not in a comment per file.

**Review by ljharb — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (armano2) [reply package.json]: ok, previously `expect` was not always imported, now it should be consistent

File: __tests__/src/util/isNonInteractiveRole-test.js (hunk 2 above)
Comment (armano2) [reply __tests__/src/util/isNonInteractiveRole-test.js]: done

**Comment by jessebeach** [comment]
@ljharb looks like your requested changes were made. Shall we get this landed?

**Review by jessebeach — APPROVED** [review]
lgtm, thanks @armano2 !!

**Review by ljharb — APPROVED** [review]

**Pull request merged**